snapshots/unicef/2023-06-16/diarrhea.xlsx.dvc

```diff
@@ -0,0 +1,26 @@
+meta:
+  namespace: unicef
+  short_name: diarrhea
+  name: Diarrhea treatment (UNICEF, 2022)
+  version: '2023-06-16'
+  publication_year: 2022
+
+  publication_date: '2022-05-01'
+  source_name: UNICEF (2022)
+  source_published_by: Several health and multiple indicator cluster surveys compiled
```

Comment:
Shouldn't this be a longer reference to the source? Like UNICEF (2022). Diarrhoea data or something?

Reply:
Ah I thought that is the `name` component higher up? And then `source_name` is what should show in the bottom left of the chart?

Reply:
Isn't `name` the name of the dataset on Grapher admin?

Reply:
Ah yes you're right - do you mean `source_published_by` should be: UNICEF (2022). Diarrhoea data, and I should change what I currently have to `source_publisher_source` - I think?

Reply:
Oh interesting - I added it in the dvc file and it seemed to work. Let's see what happens ;)